### PR TITLE
Remove 12.3+ MPS warning from installation matrix

### PR DESCRIPTION
**Pull request by malfet**
As it's really hard to find Macs running on 12.3 in 2024

Updated stable rocm version to 5.7 and regenerated `quickstart-module.js` by running
```
./scripts/gen_quick_start_module.py > assets/quick-start-module.js
```

### CODE REVIEW

File: assets/quick-start-module.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ var archInfoMap = new Map([
   ['accnone', {title: "CPU", platforms: new Set(['linux', 'macos', 'windows'])}]
 ]);
 
-let version_map={"nightly": {"accnone": ["cpu", ""], "cuda.x": ["cuda", "11.8"], "cuda.y": ["cuda", "12.1"], "rocm5.x": ["rocm", "5.7"]}, "release": {"accnone": ["cpu", ""], "cuda.x": ["cuda", "11.8"], "cuda.y": ["cuda", "12.1"], "rocm5.x": ["rocm", "5.7"]}}
+let version_map={"nightly": {"accnone": ["cpu", ""], "cuda.x": ["cuda", "11.8"], "cuda.y": ["cuda", "12.1"], "rocm5.x": ["rocm", "5.7"]}, "release": {"accnone": ["cpu", ""], "cuda.x": ["cuda", "11.8"], "cuda.y": ["cuda", "12.1"], "rocm5.x": ["rocm", "5.6"]}}
```
Comment (albanD) [inline assets/quick-start-module.js:14]: The rocm version change is intentional (it's not in the PR title)

File: assets/quick-start-module.js [hunk 2]
```diff
@@ -248,7 +248,7 @@ $("[data-toggle='cloud-dropdown']").on("click", function(e) {
...
...with-deps-debug-2.2.0%2Bcpu.zip</a>", "stable,libtorch,windows,cuda.x,cplusplus": "Download here (Release version):<br /><a href='https://download.pytorch.org/libtorch/cu118/libtorch-win-shared-with-deps-2.2.0%2Bcu118.zip'>https://download.pytorch.org/libtorch/cu118/libtorch-win-shared-with-deps-2.2.0%2Bcu118.zip</a><br />Download here (Debug version):<br /><a href='https://download.pytorch.org/libtorch/cu118/libtorch-win-shared-with-deps-debug-2.2.0%2Bcu118.zip'>https://download.pytorch.org/libtorch/cu118/libtorch-win-shared-with-deps-debug-2.2.0%2Bcu118.zip</a>", "stable,libtorch,windows,cuda.y,cplusplus": "Download here (Release version):<br /><a href='https://download.pytorch.org/libtorch/cu121/libtorch-win-shared-with-deps-2.2.0%2Bcu121.zip'>https://download.pytorch.org/libtorch/cu121/libtorch-win-shared-with-deps-2.2.0%2Bcu121.zip</a><br />Download here (Debug version):<br /><a href='https://download.pytorch.org/libtorch/cu121/libtorch-win-shared-with-deps-debug-2.2.0%2Bcu121.zip'>https://download.pytorch.org/libtorch/cu121/libtorch-win-shared-with-deps-debug-2.2.0%2Bcu121.zip</a>", "stable,libtorch,windows,rocm5.x,cplusplus": "<b>NOTE:</b> ROCm is not available on Windows"};
```
Comment (albanD) [inline assets/quick-start-module.js:251]: Not sure what is changed here? I do see some rocm5.7 leftover though

**Review by atalman — COMMENTED** [review]
Looks ok. Please put back rocm 5.7

**Review by atalman — APPROVED** [review]

**Review by albanD — APPROVED** [review]
Thanks

**Pull request merged**